From 91b5ff92849642b41fbec217da9cf6a7d4318988 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Lu=C3=ADs=20Pereira?= <luis.artur.pereira@gmail.com>
Date: Tue, 10 Mar 2015 11:33:33 -0700
Subject: [PATCH 25/27] Make sure returned directories don't have an trailing
 '/'

We can only assure it when the postfix isn't used.
---
 xdgdirs.cpp | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/xdgdirs.cpp b/xdgdirs.cpp
index 0340958..7dcfe16 100644
--- a/xdgdirs.cpp
+++ b/xdgdirs.cpp
@@ -91,7 +91,9 @@ QString xdgSingleDir(const QString &envVar, const QString &def, bool createDir)
             qWarning() << QString("Can't create %1 directory.").arg(d.absolutePath());
     }
 
-    return d.absolutePath();
+    QString r = d.absolutePath();
+    removeEndingSlash(r);
+    return r;
 }
 
 
-- 
1.9.0

